Index: main.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/main.py b/main.py
--- a/main.py	
+++ b/main.py	
@@ -64,7 +64,7 @@
             fotos = calc.get_plan_fotos(area, orientation, cota_media, s1, s2, mf, pixel_size, focal_distance, l, q)
 
             budget_values = calc.get_plan_budget(unit_cost_foto, unit_cost_flight_hour, flight_speed, s1, s2,
-                                                 pixel_size, mf, q, l, area, fotos)
+                                                 pixel_size, mf, q, l, area)
 
             # Valores da distância total, tempo de voo e orçamento são arredondados a uma casa decimal:
 
